shark-log/src/test/java/shark/SharkLogTest.kt

```diff
@@ -0,0 +1,42 @@
+package shark
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import shark.SharkLog.Logger
+import java.io.ByteArrayOutputStream
+import java.io.PrintStream
+
+class SharkLogTest {
+
+  @Test fun `logging works when logger is set`() {
```

Comment:
This test is testing that SharkLog.d passes the result of the callback to SharkLog.logger.d, right?

Reply:
Yes, that's all it does. Not the most meaningful test, but balances the `no logger` test nicely.